### PR TITLE
Feature/5219 EM Submission Access API: Create Get Endpoint

### PR DESCRIPTION
**Pull request by sdhawan16**
## Pull Request

```
- Created view 'camdecmpsaux.vw_em_submission_access' due to complex join logic 
- Added EmSubmissionAcccessModule
- Added Get request to return data
- Note logic for status needs to be updated at a future time for with update requirements for when submissionAvailabilityCode is not REQUIRE or GRANTED
- lastWindow, submissionStatus, submissionDate, representative, and analyst were skipped for now (requirements need to be revised and updated regarding those columns)

```

### CODE REVIEW

File: src/dto/em-submission-access.params.dto.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  @IsOptional()
+  @ApiProperty()
+  @Type(() => Number)
+  quarter?: number;
+
+  @IsOptional()
+  @ApiProperty({ enum: Status })
+  status?: string;
+}
```
Comment (subrataroyCVP) [inline src/dto/em-submission-access.params.dto.ts:30]: There are no required query-param. Just want to make sure all optional.

**Review by subrataroyCVP — APPROVED** [review]
LGTM.

File: src/dto/em-submission-access.params.dto.ts (hunk 1 above)
Comment (sdhawan16) [reply src/dto/em-submission-access.params.dto.ts:30]: Yup no required parameters for the API. After talking with Jason, he wanted to keep them all optional on the API. We will require a parameter to be selected on the UI

Comment (subrataroyCVP) [reply src/dto/em-submission-access.params.dto.ts:30]: Make sense.

**Pull request merged**